Workspace.swift

```diff
@@ -31,6 +31,9 @@ configuration.proofreading.rules.remove(.autoindentResilience)
 
 configuration.documentation.localizations = ["en-US"]
 
+// https://sdggiesbrecht.github.io/Workspace/🇨🇦EN/Types/APIDocumentationConfiguration/Properties/encryptedTravisCIDeploymentKey.html
+configuration.documentation.api.encryptedTravisCIDeploymentKey = "You’ll have to encrypt one yourself and provide it here."
```

Comment:
You still need to [create a real key](https://sdggiesbrecht.github.io/Workspace/🇨🇦EN/Types/APIDocumentationConfiguration/Properties/encryptedTravisCIDeploymentKey.html) and enter it here.